framework/decode/descriptor_update_template_decoder.cpp

```diff
@@ -115,7 +115,15 @@ size_t DescriptorUpdateTemplateDecoder::Decode(const uint8_t* buffer, size_t buf
             size_t current_size = total_size;
             bytes_read +=
                 ValueDecoder::DecodeSizeTValue((buffer + bytes_read), (buffer_size - bytes_read), &struct_count);
-            if (struct_count > 0)
+            if (bytes_read == buffer_size)
```

Comment:
The written files actually had a non-zero struct_count, it was also junk (0x500000000 in the test case) because (I think) the original code hadn't initialized `acceleration_structure_khr_count`, so that also doesn't help detect the broken format.  We could try to detect  better but that would require more in-depth analysis of what was encoded before https://github.com/LunarG/gfxreconstruct/commit/7d190ac9680ee830c277871ddd8085997ccdf076 . For now I'm content to improve the warning and come back to this later if we encounter another bad capture in the wild.